packages/firestore/package.json

```diff
@@ -71,7 +71,7 @@
     "@firebase/logger": "0.2.6",
     "@firebase/util": "1.0.0",
     "@firebase/webchannel-wrapper": "0.4.1",
-    "@grpc/grpc-js": "^1.0.0",
+    "@grpc/grpc-js": "1.3.0",
```

Comment:
Do you want to use `^` version ?

Reply:
Makes sense. I didn't realize we only use fixed versions for our own dependencies.